packages/@icon-magic/cli/README.md

```diff
@@ -0,0 +1,121 @@
+# @icon-magic/cli
+
+> Icon Magic command line.
+
+## Install
+
+Using npm:
+
+```sh
+npm install --save-dev @icon-magic/cli
+```
+
+or using yarn:
+
+```sh
+yarn add @icon-magic/cli --dev
+```
+
+## Usage
+
+```
+Usage:
+  icon-magic [command] [options] <directories ...>
+
+Commands:
+  build               Construct flavors that an icon from its variants, after applying the build plugins.
+  generate            Generates the flavors of the icon in the extension types that it can be consumed.
+  distribute          Moves an icon from the source to the destination, applying plugins if specified.
+
+Options:
+  -c, --config        Path to config file name
+  -i, --inputPath     Path to the input directory of icons
+  -o, --outputPath    Path to the output directory where the generated assets are to be written to
+  -h, --help          Display usage
+  -v, --version       Display version
+```
+
+## Commands
+
+If no command is specified, the CLI runs `build` and `generate` on the
+directories specified
+
+```
+icon-magic .
+```
+
+### Build
+
+```
+icon-magic build .
+```
+
+Given a directory, finds the closest config file and generates a list of icon
+folders and the rules for the icon that will then get stored as part of the
+icon's folder.
+
+Creates all the combinations of variants for the icon.
+
+The build is responsible for going through each variant of the icon and running
+the defined set of plugins on all of them
+
+Each plugin acts as an iterator and generates themes of the same variant. It's
+responsible for creating the
```

Comment:
the...?

Reply:
Removed descriptions from here and linking to the actual packages.